### PR TITLE
Search bar UI change

### PR DESCRIPTION
**Pull request by jonathanlei**
Change DocSearch3 UI according to design.

### CODE REVIEW

**Comment by amarantha-k** [comment]
Re-running preview build job as the PR preview link was returning a 404.

**Comment by jonathanlei** [comment]
> Re-running preview build job as the PR preview link was returning a 404.

usually it takes a while for preview to show up

**Comment by amarantha-k** [comment]
@jonathanlei After selecting a search result, the page refreshes and the top menu is misaligned in the desktop view. 
https://user-images.githubusercontent.com/13425257/204924017-9e04c490-85e3-46b6-b7af-5a10ee1ed989.mov

**Comment by jonathanlei** [comment]
> @jonathanlei After selecting a search result, the page refreshes and the top menu is misaligned in the desktop view. https://user-images.githubusercontent.com/13425257/204924017-9e04c490-85e3-46b6-b7af-5a10ee1ed989.mov

@amarantha-k that's because it redirected to the current site (xrpl.org) instead of the preview site :)

**Comment by amarantha-k** [comment]
> > @jonathanlei After selecting a search result, the page refreshes and the top menu is misaligned in the desktop view. https://user-images.githubusercontent.com/13425257/204924017-9e04c490-85e3-46b6-b7af-5a10ee1ed989.mov
> 
> @amarantha-k that's because it redirected to the current site (xrpl.org) instead of the preview site :)

Oops - my bad, totally missed that!

File: styles/xrpl.css [hunk 1]
```diff
@@ -9641,6 +9641,93 @@ p {
 
 .xrp-ledger-dev-portal .DocSearch-Modal {
   top: 85px;
+  background-color: #232325;
```
Comment (mDuo13) [inline styles/xrpl.css]: This file (and the corresponding `xrpl.css.map`) shouldn't be committed to the repo; I missed that while reviewing #1581. Please delete them.

File: styles/_colors.scss [hunk 2]
```diff
@@ -13,6 +13,9 @@ $gray-700: #343437;
 $gray-800: #232325;
 $gray-900: #111112;
 $black:    #000000;
+$black-10: #F5F5F7;
+$black-40: #A2A2A4;
+$black-60: #454549;
```
Comment (mDuo13) [inline styles/_colors.scss]: These are just aliases for `$gray-100, $gray-400, and $gray-600`, right? Can we do without them? If not, I suggest explicitly defining them in terms of the original name, like this:

```suggestion
$black-10: $gray-100;
$black-40: $gray-400;
$black-60: $gray-600;
```

**Review by mDuo13 — CHANGES_REQUESTED** [review]
Can you bump the version number of the generated CSS file? If we deploy changes like this to the production site, then anyone who has the current (v12) version cached won't see the differences right away. (They'll have to refresh w/o cache, or wait for the cached version to expire.)

**Review by mDuo13 — APPROVED** [review]

**Pull request merged**